src/coreclr/jit/codegencommon.cpp

```diff
@@ -595,28 +595,6 @@ void CodeGenInterface::genUpdateRegLife(const LclVarDsc* varDsc, bool isBorn, bo
 // compHelperCallKillSet: Gets a register mask that represents the kill set for a helper call.
 // Not all JIT Helper calls follow the standard ABI on the target architecture.
 //
-// TODO-CQ: Currently this list is incomplete (not all helpers calls are
```

Comment:
We haven't done the "exact regs killed" for other helpers, so probably this comment should stay until we do it?

Reply:
Added a comment